crates/cxx-qt-gen/src/generator/rust/signals.rs

```diff
@@ -55,15 +54,12 @@ pub fn generate_rust_signals(
             std::mem::swap(&mut unsafe_call, &mut unsafe_block);
         }
 
-        let attrs = &signal.method.attrs;
-
         let fragment = RustFragmentPair {
             cxx_bridge: vec![
                 quote! {
                     #unsafe_block extern "C++" {
-                        #(#attrs)*
-                        #[rust_name = #signal_name_rust_str]
-                        #unsafe_call fn #signal_name_cpp(self: #self_type, #(#parameters),*);
+                        #[cxx_name = #signal_name_cpp_str]
```

Comment:
not sure this is worth the gain, as we still need to generate the other methods.

Something to consider later after the API shifts are complete.